docs/DeveloperGuide.adoc

```diff
@@ -14,25 +14,45 @@ endif::[]
 
 By: `Team W09-B3`      Since: `Mar 2018`      Licence: `MIT`
 
+== Introduction
+
+=== About Codeducator
```

Comment:
Good intro! I think this succinctly captures the gist of our app and communicates very well to potential developers

Reply:
The documentation for `favourites` and `dashboard` is more readable and clear. Good job!